### PR TITLE
Fix for 839

### LINKED ISSUE #839: Improve styling of sort dropdown
New design:
screen shot 2018-08-08 at 1 11 45 pm

Current app:
screen shot 2018-08-08 at 1 10 48 pm

### PR DESCRIPTION
**Pull request by maxneuvians**
close #839, sorry @sastels 😢

### CODE REVIEW

File: components/BB.js [hunk 1]
```diff
@@ -251,15 +259,19 @@ export class BB extends Component {
               <Grid item lg={8} md={8} sm={7} xs={12}>
                 <Grid container spacing={16}>
                   <Grid item xs={12} md={6}>
+                    <InputLabel className={classes.sortByLabel}>
```
Comment (sastels) [inline components/BB.js]: will screen readers associate this label with the Select? I was thinking of pulling the label out like this, but thought that was a no-no.

Comment (maxneuvians) [reply components/BB.js]: Probably not

**Comment by sastels** [comment]
You may as well get rid of `classes` in `ProfileSelector` PropTypes while you're there.

image

File: components/BB.js (hunk 1 above)
Comment (sastels) [reply components/BB.js]: eh, it passes Axe tests.

Comment (maxneuvians) [reply components/BB.js]: Adding a `for` to the label

**Comment by sastels** [comment]
Can you center the "Sort By" vertically, and blue things up a bit?
image
vs
image

File: components/BB.js [hunk 2]
```diff
@@ -259,7 +259,10 @@ export class BB extends Component {
               <Grid item lg={8} md={8} sm={7} xs={12}>
                 <Grid container spacing={16}>
                   <Grid item xs={12} md={6}>
-                    <InputLabel className={classes.sortByLabel}>
+                    <InputLabel
+                      for="sortBySelector"
```
Comment (sastels) [inline components/BB.js]: 👍

**Comment by maxneuvians** [comment]
Looks great in Chrome, don't know about  blue links, would be the only place in the app that this is blue

**Comment by sastels** [comment]
hmm, yeah, looks fine in Chrome and Safari. Just funny on Firefox.

**Review by sastels — APPROVED** [review]
ok, close enough i guess.

**Pull request merged**